catalog-spec/catalog-spec.md

```diff
@@ -119,7 +119,7 @@ The following table lists the Media Types to use for STAC structures.
 | Media Type             | Description                                                |
 | ---------------------- | ---------------------------------------------------------- |
 | `application/geo+json` | A STAC [Item](../item-spec/item-spec.md)                   |
-| `application/json`     | A STAC [Catalog](#stac-catalog-specification)              |
+| `application/json`     | A STAC [Catalog](#)              |
```

Comment:
Why not just remove the link?

Reply:
I didn't think too much about the usefulness -- I was just trying to fix the validation error. 
```suggestion
| `application/json`     | A STAC Catalog            |
```